src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.User;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
```

Comment:
Consider whether using `@BeforeAll` is the best choice here. Since `@BeforeAll` is only executed once before all tests, it might be better to use `@BeforeEach` to ensure each test has a fresh state. This is important for test independence.